src/index.ts

```diff
@@ -590,10 +590,9 @@ export class Bigtable {
     this.projectName = `projects/${this.projectId}`;
     this.shouldReplaceProjectIdToken = this.projectId === '{{projectId}}';
 
-    const handlers =
-      options.metricsEnabled === true
-        ? [new GCPMetricsHandler(options as ClientOptions)]
-        : [];
+    const handlers = !(options.metricsEnabled === false)
+      ? [new GCPMetricsHandler(Object.assign({}, options) as ClientOptions)]
+      : [];
```

Comment:
We need to send a copy of the options so that when the handler deletes the apiEndpoint the original options are not affected.